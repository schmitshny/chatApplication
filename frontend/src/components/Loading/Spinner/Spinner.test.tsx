import { describe, test, expect } from 'vitest';
import { render } from '../../../tests/test-utils';
import { Spinner } from '..';

describe('Spinner', () => {
  test('renders correctly', () => {
    const { container } = render(<Spinner />);

    expect(container).toMatchSnapshot();
  });
});
