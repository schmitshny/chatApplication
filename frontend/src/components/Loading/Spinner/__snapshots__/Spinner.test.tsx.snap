// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner > renders correctly 1`] = `
<div>
  <div
    class="sc-eCYdKt dnInPD"
  />
</div>
`;
